python/Ganga/Core/GangaRepository/SubJobXMLList.py

```diff
@@ -326,52 +331,60 @@ def __call__(self, index):
 
     def __getitem__(self, index):
         """Return a subjob based upon index"""
-        return self._getItem(index)
         try:
             return self._getItem(index)
         except Exception as err:
-            logger.error("CANNOT LOAD SUBJOB INDEX: %s" % str(index))
+            logger.error("CANNOT LOAD SUBJOB INDEX: %s. Reason: %s" % (str(index), str(err)))
```

Comment:
This doesn't need the explicit conversion with `str()`. `%s` will *always* do this call for you.